### PR TITLE
add pricing page + sales

### PR DESCRIPTION
**Pull request by carlosabadia**

### CODE REVIEW

File: docs/hosting/deploy-quick-start.md [hunk 1]
```diff
@@ -52,13 +58,7 @@ The command is by default interactive. It asks you a few questions for informati
...
 
 ```md alert info
-# Once your code is uploaded, the hosting service will start the deployment. After a complete upload, exiting from the command **does not** affect the deployment process. The command prints a message when you can safely close it without affecting the deployment.
-```
-
-The hosting service does not currently handle database or file upload operations. It is necessary to set up an external database use it within your app.
-
-```md alert info
-# If you want to deploy on a custom URL we recommend using https://redirect.pizza
+# The hosting service does not currently handle database or file upload operations. Set up an external database use it within your app.
```
Comment (Lendemor) [inline docs/hosting/deploy-quick-start.md:61]:
```suggestion
# The hosting service does not currently handle database or file upload operations. Set up an external database and use it within your app.
```

File: docs/hosting/hosting-cli-commands.md [hunk 2]
```diff
@@ -56,25 +56,8 @@ To redeploy or update your app, navigate to the project directory and type `refl
...
-  build-logs  Get the build logs for a deployment.
-  delete      Delete a hosted instance.
-  list        List all the hosted deployments of the authenticated user.
-  logs        Get the logs for a deployment.
-  regions     List all the regions of the hosting service.
-  status      Check the status of a deployment.
-"""
-)
+``` bash
+reflex deployments --help
```
Comment (Lendemor) [inline docs/hosting/hosting-cli-commands.md:60]: I thought `deployments` was replaced by `apps` ?

File: pcweb/pages/pricing/calculator.py [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+        # Total 1 month
+        rx.center(
+            rx.flex(
+                rx.badge(
+                        f"Total: ${
+                        round(
+                        BillingState.estimated_seats * BillingState.seat_rate
+                        + BillingState.estimated_ram_gb * (BillingState.mem_rate * MONTH_MINUTES)
+                        + BillingState.estimated_cpu_number * (BillingState.cpu_rate * MONTH_MINUTES)
+                        - 25
```
Comment (Kastier1) [inline pcweb/pages/pricing/calculator.py:209]: why `-25`? if this is the hobby tier we should have it marked and is the hobby tier really 25 a month?

**Review by Lendemor — CHANGES_REQUESTED** [review]

File: docs/hosting/hosting-cli-commands.md (hunk 2 above)
Comment (Kastier1) [reply docs/hosting/hosting-cli-commands.md:60]: it is for the new hosting service but i dont think these commands are relating to v1. these are still v0

tom has a pr up for the new v1 commands

File: pcweb/pages/pricing/calculator.py (hunk 3 above)
Comment (Alek99) [reply pcweb/pages/pricing/calculator.py:209]: cpu is 20 and 10$ per gig so 5

Comment (Alek99) [reply pcweb/pages/pricing/calculator.py:209]: its just the usage from the free tier

File: docs/hosting/hosting-cli-commands.md (hunk 2 above)
Comment (Alek99) [reply docs/hosting/hosting-cli-commands.md:60]: I just fixed those pages because the old codeblock was spilling over main content

Comment (Alek99) [reply docs/hosting/hosting-cli-commands.md:60]: toms pr will overide it

**Review by Kastier1 — APPROVED** [review]

**Review by picklelo — APPROVED** [review]

**Review by Lendemor — APPROVED** [review]

File: docs/hosting/hosting-cli-commands.md (hunk 2 above)
Comment (Lendemor) [reply docs/hosting/hosting-cli-commands.md:60]: understood

**Pull request merged**